lib/active_merchant/billing/gateways/braintree_blue.rb

```diff
@@ -949,13 +949,13 @@ def stored_credentials_v1(parameters, stored_credential)
         end
       end
 
-      def add_external_vault(parameters, stored_credential)
+      def add_external_vault(parameters, stored_credential, options = {})
```

Comment:
AM's pattern prefers `options` over alternate naming if it's the full options hash. I do get your point about argument 2 is from argument 3 so I'll drop the `stored_credential` one. 

Additionally the upper conditional does not need to be modified because we still need to make sure the stored credential mappings are in place before adding a NTID which is what the `return unless (stored_credential = options[:stored_credential])` logic checks.